src/IR/IR.hs

```diff
@@ -232,12 +232,13 @@ instance Comonad Expr where
   extend f e@(Prim p es _) = Prim p (fmap (extend f) es) (f e)
 
 instance Foldable Expr where
-  foldMap f (Var  _ t     ) = f t
-  foldMap f (Data _ t     ) = f t
-  foldMap f (Lit  _ t     ) = f t
-  foldMap f (App    l  r t) = foldMap f l <> foldMap f r <> f t
-  foldMap f (Let    xs b t) = mconcat (map (foldMap f . snd) xs) <> foldMap f b <> f t
-  foldMap f (Lambda _  b t) = foldMap f b <> f t
+  foldMap f (Var  _ t ) = f t
```

Comment:
As far as I can tell, this is just a formatting change (might have been introduced by mistake); please revert this

Reply:
actually nvm; if brittany decided to change this, then let's leave it like this (since brittany is supposed to be idemopotent). It'll be clobbered sooner or later.